user/caching.md

```diff
@@ -168,7 +168,7 @@ Large files that are quick to install but slow to download do not benefit from c
 
 * Travis CI fetches the cache for every build, including branches and pull requests.
 * There is one cache per branch and language version/ compiler version/ JDK version/  Gemfile location/ etc.
-* If a branch does not have its own cache yet, it uses the master branch cache.
+* If a branch does not have its own cache, Travis CI fetches the repository default branch cache.
```

Comment:
Sorry for the confusion, @plaindocs - I just double checked and it's master the one we use at the moment for pushes:

https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/shared/directory_cache/base.rb#L135